SPECS/libxml2/libxml2.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Libxml2
 Name:           libxml2
-Version:        2.11.5
+Version:        2.12.8
```

Comment:
I would request to build the dependent packages as well just to ensure we are not breaking anything.